source/plugin/assets.rst

```diff
@@ -59,3 +59,16 @@ following code:
     ``getResourceAsStream`` methods in ``Class``\es and ``ClassLoader``\s. **These methods should generally be avoided**
     within the SpongeAPI environment in favor of the ``AssetManager`` in order to provide a more confluent way of
     retrieving resources not only within your own plugin, but for other plugins as well.
+
+Working with Assets
```

Comment:
This headline exists already.

Reply:
> IMO this section should be better moved to the config section. From the context it does not match here.
> 
> Maybe update this page:
> https://docs.spongepowered.org/stable/en/plugin/configuration/loaders.html?highlight=config#example-loading-a-default-config-from-the-plugin-jar-file

Yeah I can see It making sense that way. Put it in assets originally as assets explanation came after but that link shows assets are linked prior and an example is shown so they dev should understand by then. Newest commit will reflect those changes. And fixed that header issue; thought I had changed that earlier honestly